### DISCUSSION TITLE
Adding "auto" as one of the action: types for Boostagrams

### DISCUSSION THREAD

**Discussion by dhk2** [Podcastindex-org/podcast-namespace#500] [Enhancement Proposal]
Current spec:
    action recommended: (str) "boost" or "stream"
If changed:
   action recommended: (str) "boost" or "stream" or "auto"

Signifying an automatically initiated payment unlike a user initiated boost or player initiated stream payment. 

Episode specific information should be optional, presence indicating a per-episode patronage instead of a time frame patronage donation.

The message field may have a human readable string relaying something about the automatic payment.  
- patronage for 4/23
- patronage for 5/23 gifted from dave@dave.social
- gifted thanks for releasing "awesome podcast title"
- scheduled payday patronage boost  
- podpooldonationplayer weekly split for awesome podcast

**Comment by adamc199** [comment]
Since the term 'subscription' has always been confusing, let's think of some other terms that are more accurate:

Timed
Auto
Scheduled

I think this is a great idea.

Especially since split recipients like app devs will also be able to count on some sustainable income.

**Reply by dhk2** [reply]
Yeah, there are definitely connotation issues with 'subscription' 
I also considered:
- recurring
- patronage
- automatic

**Reply by daveajones** [reply]
“auto” sticks out to me. Simple.

**Reply by brianoflondon** [reply]
I'd go for 'auto' on the grounds of fewest characters.

**Reply by dhk2** [reply]
"auto" does have the advantage of being pithy. Makes sense thematically as well. Boosts are directly initiated by the booster, Stream payments are initiated by the media player in response to payer playing the content. Auto payments are automatically generated independent of the payers current activity.

**Comment by dhk2** [comment]
Thanks Dave!
